### DISCUSSION TITLE
Welcome to monero-docs Discussions!

### DISCUSSION THREAD

**Discussion by luigi1111** [monero-project/monero-docs#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by detherminal** [comment]
Please don't make this repo work insanely slow as monero-site. Please.

**Comment by luigi1111** [comment]
Who's going to work on it? Is there a suitable maintainer available? Otherwise I'll merge things if/when they show up.

**Reply by detherminal** [reply]
I can, if you want to.

**Reply by luigi1111** [reply]
Invited you.

**Comment by nahuhh** [comment]
Why are we using github discussions?
https://matrix.to/#/#monero-docs:monero.social

**Comment by nahuhh** [comment]
And @detherminal .. why r u committing stuff directly to the repo 🧐

supposed 2 fork monerodocs.org and continue with prs

@luigi1111 I told you I'd maintain this a long time ago.. 🤷‍♂️

**Reply by luigi1111** [reply]
@detherminal Please join the IRC room and work out with interested parties how to proceed. You were the only one who showed up on Github but apparently there's more interested parties. I'm going to pause your perms until the budding "docs workgroup" elects a maintainer.
